### ISSUE TITLE
PyCrytoDome is not being recognized

### ISSUE THREAD

**Issue by KuttKatrea** [bndr/pipreqs#66]
I have a project where I use pycryptodomex, but pipreqs don't identify it as a dependency

**Comment by jonafato** [comment]
This package likely just needs a mapping defined. If someone opens up a PR to do so and fix this issue, I'll merge it.

**Comment by answerquest** [comment]
Hi, my PR for replacing pycrypto just got merged, but I actually started using PyCryptodome**X** in my own application like @KuttKatrea , because being a completely separate package using the parent name `Cryptodome` this is what builds successfully when compiling my project into a binary using pyinstaller. And it works fine on heroku too.

```python
from Cryptodome.PublicKey import RSA #uses pycryptodomex package.. disambiguates from pycrypto, pycryptodome
```
Ref:  
- https://github.com/pyinstaller/pyinstaller/issues/2365
- https://stackoverflow.com/a/50009769/4355695

And I still have to manually add it to my requirements.txt after running pipreqs, as the original issue here says.  
So, apologies @KuttKatrea  for misreading your original issue, didn't notice the x back then. Will see if I can get pycryptodomex in also.
